app/services/messages/markdown_renderer_service.rb

```diff
@@ -0,0 +1,294 @@
+class Messages::MarkdownRendererService
+  CHANNEL_RENDERERS = {
+    'Channel::Email' => :render_html,
```

Comment:
does it change the content in the webhook. Webhook content should be markdown only.